webapp/app/sensors/templates/sensors.html

```diff
@@ -42,20 +42,27 @@ <h3 id="header">{{ loc_message }}</h3>
                     <th>Device ID</th>
                     <th>Name</th>
                     <th>Last updated</th>
-                    <!-- <th>Shelf</th> -->
+                    <th>Zone</th>
+                    <th>Aisle</th>
+                    <th>Column</th>
+                    <th>Shelf</th>
                     <th class="column-title no-link last"><span class="nobr">Action</span>
                   </tr>
                 </thead>
                 
                 <tbody>
                     {% for t_sensor in sensors %}
                     <tr class='clickable-row' method = POST>
-                      <td>{{ t_sensor.sensors_id }}</td>
-                      <td>{{ t_sensor.sensor_types_sensor_type }}</td>
-                      <td>{{ t_sensor.sensors_device_id }}</td>
-                      <td>{{ t_sensor.sensors_name }}</td>
-                      <td>{{ t_sensor.sensors_last_updated }}</td>
-                      <td class=" last"><a href="/sensors/sensor_form?query={{ t_sensor.sensors_id }}">Edit</a>
+                      <td>{{ t_sensor.id }}</td>
```

Comment:
I think I remember trying to help Flora debug this at some point, and she was seeing different behaviour between her local setup and the deployed version, where one had e.g. "sensor.sensors_id" and the other had "sensor.id".   We were speculating back then that this was due to different sqlalchemy versions...     
Is this understood now?

Reply:
To the degree that I can confirm that it indeed is because of sqlalchemy versions. I haven't fully understood where the change happened.